docs/sync.js

```diff
@@ -547,29 +547,52 @@ class ApiKeyAuth {
     disable(id) { }
 }
 
+/**
+ * The type of an authentication provider, used to authenticate a user.
+ * @typedef Realm.App.Sync~ProviderType
+ * @type {("anon-user"|"api-key"|"local-userpass"|"custom-function"|"custom-token"|"oauth2-google"|"oauth2-facebook"|"oauth2-apple")}
+ */
+
+/**
+ * Describes an error when an incompatible synced Realm is opened. The old version of the Realm can be accessed in readonly mode using the configuration() member
```

Comment:
This is the description of `IncompatibleSyncedRealmError`. We need a correct description of `UserIdentity`.

Reply:
Thanks!